tests/observability/metrics/utils.py

```diff
@@ -741,6 +750,7 @@ def get_pvc_size_bytes(vm: VirtualMachineForTests) -> str:
                 PersistentVolumeClaim(
                     name=vm_dv_templates[0].metadata.name,
                     namespace=vm.namespace,
+                    client=client,
```

Comment:
```suggestion
                    client=vm.client,
```
no need to pass client param to function at all this way